xml/System/Memory`1.xml

```diff
@@ -22,9 +22,34 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of items in the <see cref="T:System.Memory`1" />. </typeparam>
+    <summary>Represents a contiguous region of memory. </summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+  
+Like <xref:System.Span%601>, `Memory<T>` represents a contiguous region of memory. Unlike <xref:System.Span%601>, however, `Memory<T>` is not a [ref struct](~/doocs/csharp/reference-semantics-with-value-types?view=netcore-2.1.md#ref-struct-type). This means that `Memory<T>` can be backed by:
```

Comment:
> This means that...

That difference, struct vs ref struct, really just impacts whether or not it can be stored on the heap, which in turn impacts whether it can be used as a field in a class, whether it can used across await and yield boundaries, etc.